### PR TITLE
Use iree-dist tarball instead installing via pip

### PR DESCRIPTION
**Pull request by marbre**

### CODE REVIEW

File: build_tools/install_iree_host_tools.sh [hunk 1]
```diff
@@ -8,27 +8,78 @@
 
 # Bash script to build / install the IREE host tools.
 
-if [ "${VIRTUAL_ENV}" != "" ]; then
+if [[ $# -ne 0 && $# -ne 1 ]] ; then
+  echo "Usage: $0"
+  echo "   or: $0 -pip"
+  echo ""
+  echo "By default, the iree-dist installed is fetched and unpacked."
```
Comment (lucas-camp) [inline build_tools/install_iree_host_tools.sh:15]:
```suggestion
  echo "By default, the iree-dist tarball is fetched and unpacked."
```

File: build_tools/install_iree_host_tools.sh [hunk 2]
```diff
@@ -8,27 +8,78 @@
...
+mkdir -p build-iree-host-install-${IREE_VERSION}/bin
+ln -sfn build-iree-host-install-${IREE_VERSION} build-iree-host-install
+
+
+# Use the iree-dist tarball
+
+if [[ $# -eq 0 ]] ; then
+  python3 -m venv venv-iree
+  source venv-iree/bin/activate
+  pip3 install -r numpy Pillow
```
Comment (marbre) [inline build_tools/install_iree_host_tools.sh:59]: The `-r` needs to be dropped here. Will adjust this together with incorporating your review suggestions.

File: build_tools/install_iree_host_tools.sh (hunk 1 above)
Comment (marbre) [reply build_tools/install_iree_host_tools.sh]: Done.

**Comment by marbre** [comment]
I revised the patches and splited `requirements.txt` into `requirements.txt` and `requirements-compiler.txt`.

**Review by lucas-camp — APPROVED** [review]

**Pull request merged**